source/tutorials/beginner.rst

```diff
@@ -159,7 +159,7 @@ Reading a .txt File
 In building your first Python script we will set up our workspace, read a
 :code:`.txt` file, and learn git fundamentals.
 
-Open a terminal to begin:
+Open a regular or anaconda/miniconda terminal to begin:
```

Comment:
```suggestion
Open a terminal application to begin (on Linux or Mac, open **Terminal**; on Windows, open **Anaconda Prompt**):
```

Reply:
I already edited this section based on your comment in the issue, so now the suggested change cannot be merged (but I feel it is redundant with my change -- adding this info as a note).